x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/build_events_query.ts

```diff
@@ -74,25 +74,40 @@ export const buildEventsScrollQuery = ({
       },
     },
   ];
-  return {
+  const searchQuery = {
     allowNoIndices: true,
     index,
-    scroll,
     size,
     ignoreUnavailable: true,
     body: {
       query: {
         bool: {
           filter: [
-            ...filterWithTime,
+            ...[filterWithTime],
             {
               match_all: {},
             },
           ],
         },
       },
       track_total_hits: true,
```

Comment:
I don't want to use track_total_hits but search_after needs it in order to know when to stop searching after. Without it, search_after seems to run forever I think because events are coming in faster than the search result can resolve, so searching for the past five minutes, after each search, more data is sent in and the query searches for the next 5 minutes forever. Would love some comments on how to maybe fix this either in the search query or how to utilize search_after more like scroll? I still feel scroll is the proper way to query for these events but until the performance issues are fixed I think we have to use search_after.